characters/player/bullet.gd

```diff
@@ -0,0 +1,21 @@
+extends Area2D
+
+var travelled = 0
+const DAMAGE = 5
+
+func _physics_process(delta):
+	const SPEED = 1000
```

Comment:
these constants declaration should be moved out of the function to become member variables